src/blas/backends/cublas/cublas_level1.cpp

```diff
@@ -32,8 +32,8 @@ namespace column_major {
 
 // Level 1
 template <typename Func, typename T1, typename T2>
-inline void asum(Func func, cl::sycl::queue &queue, int64_t n, cl::sycl::buffer<T1, 1> &x,
-                 const int64_t incx, cl::sycl::buffer<T2, 1> &result) {
+inline void asum(const char *func_name, Func func, cl::sycl::queue &queue, int64_t n,
```

Comment:
I think we can avoid changing the function signatures if we pass an explicit string to `CUBLAS_ERROR_FUNC_T`.